Editor/Window/ConnectToFleetInput.cs

```diff
@@ -142,9 +144,20 @@ private async Task UpdateFleetMenu()
                 {
                     _fleetAttributes = new List<FleetAttributes>(); 
                 }
+
                 _fleetNameDropdownContainer.choices = _fleetAttributes.Select(fleet => fleet.Name).ToList();
                 _fleetNameDropdownContainer.value = _stateManager.AnywhereFleetName;
                 _fleetIdText.text = _stateManager.AnywhereFleetId;
+
+                var fleet = _fleetAttributes.FirstOrDefault(fleet => fleet.Name == _stateManager.AnywhereFleetName);
+                if (fleet != null)
+                {
+                    var textProvider = new TextProvider();
+                    _statusIndicator.Set(State.Success,
+                        textProvider.Get(fleet.Status == Amazon.GameLift.FleetStatus.ERROR
```

Comment:
If the fleet goes into ERROR, should the state be `State.Success` ? Seems like the status should also be ERROR right?

Reply:
Updated!